src/components/TodoContainer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import TodosList from './TodosList';
+import Header from './Header';
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Setup development Environment for you're Mac",
+        completed: false,
+      },
+      {
+        id: 2,
+        title: 'Create dependencies: for example- prettier',
+        completed: false,
+      },
+      {
+        id: 3,
+        title: 'Deploy website',
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    console.log('clicked', id);
+  };
```

Comment:
- [ ]  Inside your TodoContainer, the handleClick function doesn't up the state of the component, and this causes the checkbox to be inactive even when it is clicked on.

An option is to use the `this.setState()` anytime `handleClick()` is called to update the right object

Reply:
Thank you for your kind review and methods to improve!

Reply:
I have implemented the changes. TodoContainer now updated to allow state change of boolean on check.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/81635839/165299180-05e58573-1b11-4860-958f-6a221a50dcc0.png">